src/components/CVE-2022-1388/CVE-2022-1388.tsx

```diff
@@ -0,0 +1,241 @@
+            .then((isAvailable) => {
+                setIsCommandAvailable(isAvailable); // Set the command availability state
+                setOpened(!isAvailable); // Set the modal state to opened if the command is not available
+                setLoadingModal(false); // Set loading to false after the check is done
+            })
+            .catch((error) => {
+                console.error("An error occurred:", error);
+                setLoadingModal(false); // Also set loading to false in case of error
+            });
+    }, []);
+
+    /**
+     * Handles incoming data from a child process and appends it to the current output state.
+     * @param {string} data - The string data recieved from the child process.
+     */
+    const handleProcessData = useCallback((data: string) => {
+        setOutput((prevOutput) => prevOutput + "\n" + data); // Append new data to the previous output.
+    }, []);
+    /**
+     * handleProcessTermination: Callback to handle the termination of the child process.
+     * Once the process termination is handled, it clears the process PID reference and
+     * deactivates the loading overlay.
+     * @param {object} param - An object containing information about the process termination.
+     * @param {number} param.code - The exit code of the terminated process.
+     * @param {number} param.signal - The signal code indicating how the process was terminated.
+     */
+    const handleProcessTermination = useCallback(
+        ({ code, signal }: { code: number; signal: number }) => {
+            // If the process was successful, display a success message.
+            if (code === 0) {
+                handleProcessData("\nProcess completed successfully.");
+                // If the process was terminated manually, display a termination message.
+            } else if (signal === 15) {
+                handleProcessData("\nProcess was manually terminated.");
+                // If the process was terminated with an error, display the exit and signal codes.
+            } else {
+                handleProcessData(`\nProcess terminated with exit code: ${code} and signal code: ${signal}`);
+            }
+            // Clear the child process pid reference. There is no longer a valid process running.
+            setPid("");
+            // Cancel the Loading Overlay. The process has completed.
+            setLoading(false);
+        },
+        [handleProcessData]
+    );
+
+    /**
+     * Handles form submission, executes the exploit command, and updates the output state.
+     *
+     * @param values - The form values containing host IP, target IP, and host port.
+     */
+    const onSubmit = async (values: FormValuesType) => {
+        setLoading(true);
+
+        // Disallow saving until the tool's execution is complete
+        setAllowSave(false);
+
+        // Define the command arguments to be executed
+        const args = [
+            "./exploits/CVE-2022-1388.py",
+            "-h",
```

Comment:
Need to change the option from -h to something like -H.

```suggestion
            "-H",
```

Reply:
changed paramter